src/components/InlineCodeBlock/index.js

```diff
@@ -9,7 +8,7 @@ const InlineCodeBlock = ({
     textStyle,
 }) => (
     <TDefaultRenderer
-        style={{...boxModelStyle, ...textStyle, ...styles.dInlineBlock}}
```

Comment:
I don't think this style is valid on iOS or Android. I removed it and didn't see a change but lmk if you do @shawnborton

Reply:
Oh wait nevermind this is going into the HTML renderer... that's super confusing actually. I think we should maybe place these styles somewhere else since there are styles acceptable to use in "react native" and then styles that can be parsed by this engine 😬

Reply:
Actually nvm again. This doesn't seem like it does anything at all